public/home.html

```diff
@@ -416,8 +416,14 @@ <h2 class="info-header">GPA Type and Switching Quarters</h2>
       GPA for each quarter, which when clicked allows you to see your grades and
       assignments for that quarter.
       </p>
+
+      <hr>
+      <p class="info-text" id="updates-container"><span id="updates"></span></p>
+      <p id="moreVersions"><a  href="https://github.com/Aspine/aspine/blob/master/CHANGELOG.md">&lt&lt See Older Versions of Aspine &gt&gt</a></p>
```

Comment:
Remove extraneous space between `a` and `href` and prevent overlong lines; use
kebab-case instead of camelCase for CSS ID
```suggestion
      <p id="more-versions">
        <a href="https://github.com/Aspine/aspine/blob/master/CHANGELOG.md">
          &lt&lt See Older Versions of Aspine &gt&gt
        </a>
      </p>
```